cocos/tiledmap/assembler/simple.ts

```diff
@@ -74,9 +74,6 @@ class Simple implements IAssembler {
             const batcher = director.root!.batcher2D;
             _accessor = new StaticVBAccessor(device, vfmtPosUvColor);
             //batcher.registerBufferAccessor(Number.parseInt('TILED-MAP', 36), _accessor);
-            director.on(DirectorEvent.BEFORE_DRAW, () => {
-                _accessor.reset();
-            });
```

Comment:
Will this removal break user code?

Reply:
Actually not, because tilemap has multiple TiledLayers, and each TiledLayer is not updated simultaneously, so the index needs to be cleared before each rendering. I have modified the filling method and there is no need to reset it again